spring-integration-smb/src/test/resources/META-INF/spring/integration/SmbTests-context.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
Can we somehow go without an XML configs from now on if there is no strict requirements to test exactly our XML components?

I see that you have only an `SmbSessionFactory` over here, but that doesn't really justify a whole `ApllicationContext` just for this.
The `SmbSessionFactory` can be created manually and we can control its lifecycle in the test ourselves.

Reply:
I originally just used constants in the class that needed to be edited with real values... I figured this was a cleaner approach.

Reply:
When we switch to Testcontainers, it is going to be OK to use those "real values" in the constant. 
See my approach for MQTT - `MosquittoContainerTest`. That's exactly what I'd like to see for SMB tests against Samba in Docker container.
I just have no idea how that hard going to be to use that Docker image and initialize it properly for our use-cases...